app/assets/javascripts/discourse/app/components/fast-edit.gjs

```diff
@@ -53,6 +53,13 @@ export default class FastEdit extends Component {
       const result = await ajax(`/posts/${this.args.post.id}`);
       const newRaw = result.raw.replace(this.args.initialValue, this.value);
 
+      // Warn the user if we failed to update the post
+      if (newRaw === result.raw) {
```

Comment:
Added this to show an error message instead of "nothing" when we fail to update the post (for whatever reason).